api/src/main/java/org/pmiops/workbench/mail/MailService.java

```diff
@@ -12,6 +12,15 @@
   void sendWelcomeEmail(final String contactEmail, final String password, final String username)
       throws MessagingException;
 
+  void sendWelcomeEmail(
```

Comment:
I may be missing context, but I was wondering, would it be possible to consolidate these two sendWelcomeEmail methods? I wonder if passing the User and and DbUser objects would allow you to do this, and then the logic of determining which type of email to send could  be shifted from the ProfileController to the MailService.

Reply:
so the existing sendWelcomeEmail will be deleted later once Controlled tier is enabled on the environment. I will add comments